wgetpaste

```diff
@@ -409,10 +422,15 @@ per user in any of ~/.wgetpaste.conf or ~/.wgetpaste.d/*.conf.
 
 An additional http header can be passed by setting HEADER_\${SERVICE} in any of the
 configuration files mentioned above. For example, authenticating with github gist:
-HEADER_gists="Authorization: token 1234abc56789..."
+HEADER_gists="Authorization: token 1234abc56789...", or with gitlab snippets:
+HEADER_snippets="PRIVATE-TOKEN: 1234abc56789..."
+
+You can also set PUBLIC_gists='false' if you want to default to secret instead of
+public github gists. In the case of gitlab, you can set VISIBILITY_snippets= to
+'public', 'private' or 'internal'"
 
-In the case of github gist you can also set PUBLIC_gists='false' if you want to
-default to secret instead of public gists.
+To change your gitlab server, you can override the default API URL setting
+URL_snippets='https://gitlab.[server].com/api/[version]/snippets'
```

Comment:
I think we want to hardcode v4 here since we can't guarantee that it'll keep working with newer API versions, no?

Reply:
You're right!